tests/ui/documents/test_documents_filter.py

```
@@ -258,3 +259,179 @@ def test_get_first_isbn():
+    """Test get_cover_art with multiple ISBNs."""
+    record = {
+        "identifiedBy": [
+            {"type": "bf:Isbn", "value": "9782222222222"},
+            {"type": "bf:Isbn", "value": "9781111111111"},
+            {"type": "bf:Isbn", "value": "9783333333333"},
+        ]
+    }
+
+    with mock.patch("rero_ils.modules.documents.views.get_thumbnail_url") as mock_get_thumbnail:
+        # First ISBN returns None, second returns a URL
+        def side_effect(isbn, cached=True):
```

Ruff (0.15.1)

[warning] Unused function argument: `cached`

(ARG001)